CHANGELOG.md

```diff
@@ -5,10 +5,10 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 Note: This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 All notable changes to this project will be documented in this file.
-## [1.1.1] - 2020-05-03
-- Replace hobo_support with invoca_utils
 
-[1.1.1]: https://github.com/Invoca/exceptional_synchrony/compare/v1.1.0...v1.1.1
+## [1.3.0] - UNRELEASED
+### Added
+- Extend `EMP.defer` to have a new keyword argument (`wait_for_result: false`) in the event that the caller does not want to be blocked until the background thread returns. To preserve existing behavior, by default `EMP.defer` will block in order to return the value (or raise an exception) from the deferred block.
```

Comment:
I was a bit confused by `: false` since that implied to me that `false` was the default. Can you flip that to `: true` and then say callers can pass `wait_for_result: false` if they want no blocking?

Reply:
Good call, I have reworded this to remove the initial `: false` and be clear what the default is